src/styles/styles.js

```diff
@@ -2340,6 +2340,11 @@ const styles = {
         backgroundColor: colors.black,
         flex: 1,
     },
+    passwordTextHint: {
+        lineHeight: 18,
+        marginBottom: 4,
+        marginTop: 4,
+    },
```

Comment:
let's rename it to `formHelp` and adopt all the styles from formError except the color. Place it before the `formError` style.
Change the color to the one that is used on this text.
it would look like
```
    formHelp: {
         color: themeColors.textSupporting,
        fontSize: variables.fontSizeLabel,
        lineHeight: 18,
        marginBottom: 4,
    },
```
    
remove `styles.textLabelSupporting` and keep the `styles.mt1`.

Reply:
this style is a unique override for the password input, that is why I chose passwordTextHint. Maybe passwordTextSupporting would be better? as it's a variant of this. 
fromHelp sounds to me to be generic for re-use, like textLabelSupporting.

Reply:
We can try to generalize this. 
Why? 
1. There could be other places where we want the help text like this.

Reply:
ok :)